server/src/main/java/org/elasticsearch/transport/ProxyConnectionStrategy.java

```diff
@@ -226,9 +218,14 @@ public void onFailure(Exception e) {
     }
 
     private NoSeedNodeLeftException getNoSeedNodeLeftException(Collection<Exception> suppressedExceptions) {
-        final var e = new NoSeedNodeLeftException(
-            "Unable to open any proxy connections to cluster [" + clusterAlias + "] at address [" + address.get() + "]"
+        final var msg = Strings.format(
+            "Unable to open any proxy connections to cluster [%s] at address [%s], configuredAddress=[%s], configuredServerName=[%s]",
+            clusterAlias,
+            address.get(),
+            configuredAddress,
+            configuredServerName
         );
+        final var e = new NoSeedNodeLeftException(msg);
```

Comment:
The PR's description says "move .. to log msg". But do we log this error on the server side? If so, can we have a test for it?